doc/how-to/replication/repl_leader_elect.rst

```diff
@@ -20,12 +20,11 @@ Configuration
         election_timeout = <seconds>,
         replication_timeout = <seconds>,
         replication_synchro_quorum = <count>,
-        election_fencing_enabled = <boolean>
+        election_fencing_mode = <string>
     })
 
-*   ``election_mode`` -- specifies the role of a node in the leader election
```

Comment:
Why this was removed? Seems unrelated to fencing option.

Reply:
Ooh, thanks - deleted by mistake. Thanks for catching that.